exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/SQLiteDialect.kt

```diff
@@ -156,5 +158,19 @@ open class SQLiteDialect : VendorDialect(dialectName, SQLiteDataTypeProvider, SQ
     companion object {
         /** SQLite dialect name */
         const val dialectName: String = "sqlite"
+        val ENABLE_UPDATE_DELETE_LIMIT by lazy {
```

Comment:
Please move that lazy val inside SQLiteDialect as it's possible to connect to different SQLite instances within a single application.

Reply:
Although SQLiteDialect may connect to different SQLite instances, it uses the same sqlite-jdbc driver. The value of this val only depends on the version of sqlite-jdbc application used.